README/Data/Nat.agda

```diff
@@ -12,31 +12,34 @@ module README.Data.Nat where
 -- The natural numbers and various arithmetic operations are defined
 -- in Data.Nat.
 
-open import Data.Nat
+open import Data.Nat using (ℕ; _+_; _*_)
 
+-- _*_ has precedence 7 over precedence 6 of _+_
+-- precedence of both defined in module Agda.Builtin.Nat
 ex₁ : ℕ
-ex₁ = 1 + 3
+ex₁ = 1 + 3 * 4
 
 -- Propositional equality and some related properties can be found
 -- in Relation.Binary.PropositionalEquality.
 
-open import Relation.Binary.PropositionalEquality
+open import Relation.Binary.PropositionalEquality using (_≡_; refl)
 
 ex₂ : 3 + 5 ≡ 2 * 4
 ex₂ = refl
 
 -- Data.Nat.Properties contains a number of properties about natural
 -- numbers.
 
-import Data.Nat.Properties as Nat
+import Data.Nat.Properties as Nat using (*-comm; +-identityʳ)
 
 ex₃ : ∀ m n → m * n ≡ n * m
 ex₃ m n = Nat.*-comm m n
 
 -- The module ≡-Reasoning in Relation.Binary.PropositionalEquality
 -- provides some combinators for equational reasoning.
 
-open ≡-Reasoning
+open Relation.Binary.PropositionalEquality using (cong; module ≡-Reasoning)
+open ≡-Reasoning using (begin_; _≡⟨_⟩_; _∎)
```

Comment:
Hmm I'm wondering if qualifying import statements for reasoning and solver modules is perhaps overkill? I certainly don't want to have to do so every time I use them in the main library...

Reply:
I see you point.
It took me some time to figure out where the constructors and operators exactly come from and what do they exactly mean, so I guess for someone that is starting with the stdlib from the README files makes sense, but perhaps this must not be extensively applied to the library.

Reply:
Okay, fair enough, that's a good argument.